cmd/messages_text.go

```diff
@@ -280,6 +284,17 @@ func (mt *MessagesText) _select(name string) {
 	mt.ScrollToHighlight()
 }
 
+func onHighlighted(added, removed, remaining []string) {
```

Comment:
Please convert this into a receiver function:
```go
func (mt *MessagesText) onHighlighted(...) {}

Reply:
I was actually trying this at first but failed because I'm new to programming in go. Understood what was missing today after some sleep.